core/contracts/financial_templates/implementation/TokenFactory.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.5.0;
+import "./Token.sol";
+import "../interfaces/TokenInterface.sol";
+
+/**
+ * @notice A burnable and mintable ERC20.
+ */
```

Comment:
nit: copy and paste error? This seems to be the wrong comment for this class.

Reply:
yep